### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by stevejthorpe**
Created a basic react-native 'hello world' and edited the README.

### CODE REVIEW

**Comment by qaiser42** [comment]
@stevejthorpe i will setup my local dev environment and try the hello world app out

**Review by qaiser42 — CHANGES_REQUESTED** [review]
can you please add a .gitignore file to untrack temp files?

**Comment by qaiser42** [comment]
> found 4 low severity vulnerabilities
>   run `npm audit fix` to fix them, or `npm audit` for details

if it's easy to fix, then please go for it :)

**Comment by qaiser42** [comment]
can you also please .gitignore `package-lock.json`? thx

**Comment by stevejthorpe** [comment]
npm audit and .gitignore added (not sure why I missed it).

**Review by stevejthorpe — COMMENTED** [review]
How do I approve this change?

**Comment by qaiser42** [comment]
^awesome, thx

File: README.md [hunk 1]
```diff
@@ -1 +1,139 @@
-# AsylumAdvice app
+<!-- PROJECT SHIELDS --> <!-- *** I'm using markdown "reference style" links for readability. *** Reference links are enclosed in brackets [ ] instead of parentheses ( ). *** See the bottom of this document for the declaration of the reference variables *** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use. *** https://www.markdownguide.org/basic-syntax/#reference-style-links -->
+
+[![Contributors][contributors-shield]][contributors-url] [![Forks][forks-shield]][forks-url] [![Stargazers][stars-shield]][stars-url] [![Issues][issues-shield]][issues-url] [![MIT License][license-shield]][license-url] [![LinkedIn][linkedin-shield]][linkedin-url]
+
+<!-- PROJECT LOGO --> <br>
+
+[![Logo](public/images/asylum-advice-logo.svg)](https://github.com/AsylumAdvice/app) # Asylum Advice Asylum Advice helps asylum seekers find the information that is relevant to their asylum procedure.<br>
```
Comment (qaiser42) [inline README.md]: Screenshot 2020-03-03 at 11 31 29

nit pick: this is currently weirdly rendered in markdown

**Comment by qaiser42** [comment]
can you maybe please also refer to https://reactnative.dev/docs/getting-started so that new developers know what tools they have to install first?

**Comment by qaiser42** [comment]
btw which template did you used for this hello-world project `expo-template-blank-typescript`?

File: README.md (hunk 1 above)
Comment (stevejthorpe) [reply README.md]: I used the cli create-react-app, not expo.

Comment (stevejthorpe) [reply README.md]: I mean to say, the cli react-native init, not expo.

Comment (qaiser42) [reply README.md]: the documentation says `Follow these instructions if you need to build native code in your project. For example, if you are integrating React Native into an existing application, or if you "ejected" from Expo, you'll need this section.`

i think for our app expo would be sufficient. when i tried running `npm start` on this hello-world example i got the following log

```
 npm start

> app@0.0.1 start /Users/qaiserabbasi/git/app
> react-native start

┌──────────────────────────────────────────────────────────────────────────────┐
│                                                                              │
│  Running Metro Bundler on port 8081.                                         │
│                                                                              │
│  Keep Metro running while developing on any JS projects. Feel free to        │
│  close this tab and run your own Metro instance if you prefer.               │
│                                                                              │
│  https://github.com/facebook/react-native                                    │
│                                                                              │
└──────────────────────────────────────────────────────────────────────────────┘

Looking for JS files in
   /Users/qaiserabbasi/git/app

Loading dependency graph, done.

To reload the app press "r"
To open developer menu press "d"
```

there's no QR code popping up / there's nothing happening after this message. when i used `expo` i got a nice overview page with a QR code to scan and everything works perfectly

Screenshot 2020-03-03 at 11 54 44

Comment (stevejthorpe) [reply README.md]: ok, so shall I re-create with expo and make another pull request?

**Comment by qaiser42** [comment]
Yea please. I think creating a new one would be better. Merci

--

Mit freundlichen Grüßen / Best regards
B.Sc. Computer Science

Qaiser Abbasi | Freiberuflicher IT-Consultant

Scala • Elasticsearch • BigData • DevOps • Microservice


Mobil: ‭+49 1577 4080732

--------------------------------------------

Truth is ever to be found in simplicity, and not in the multiplicity and
confusion of things. – Newton

**Pull request closed without merging**